docs/LINUX.md

```diff
@@ -0,0 +1,121 @@
+```
+
+# Usage
+
+You can start sending logs by setting the [ingestion key][ingestion-key]:
+
+```shell script
+logdna-agent -k <YOUR INGESTION KEY>
+```
+
+The agent expose short argument abbreviations for commonly used options (`-k`, `-t`, `-d` and `-c`).
+
+You can use `--help` flag to list all the command and environment variable options. For example with agent
+version `3.3.0`:
+
+```shell script
+$ logdna-agent --help
+LogDNA Agent 3.3.0
+A resource-efficient log collection agent that forwards logs to LogDNA.
+
+USAGE:
+    logdna-agent [OPTIONS]
+
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+    -c, --config <config>
+            The config filename [env: LOGDNA_CONFIG_FILE=]  [default: /etc/logdna/config.yaml]
+
+        --db-path <db-path>
+            The directory the agent will store it's state database. Note that the agent must have write access to the
+            directory and be a persistent volume. Defaults to "/var/lib/logdna-agent/" [env: LOGDNA_DB_PATH=]
+        --endpoint-path <endpoint-path>
+            The endpoint to forward logs to. Defaults to "/logs/agent" [env: LOGDNA_ENDPOINT=]
+        --exclude-regex <exclusion-regex>...
+            List of regex patterns to exclude files from monitoring [env: LOGDNA_EXCLUSION_REGEX_RULES=]
+        --exclude <exclusion-rules>...
+            List of glob patterns to exclude files from monitoring, to add to the default [env: LOGDNA_EXCLUSION_RULES=]
+        --gzip-level <gzip-level>
+            If compression is enabled, this is the gzip compression level to use. Defaults to 2 [env:
+            LOGDNA_GZIP_LEVEL=]
+        --host <host>
+            The host to forward logs to. Defaults to "logs.logdna.com" [env: LOGDNA_HOST=]
+        --include-regex <inclusion-regex>...
+            List of regex patterns to include files from monitoring [env: LOGDNA_INCLUSION_REGEX_RULES=]
+        --include <inclusion-rules>...
+            List of glob patterns to includes files for monitoring, to add to the default (*.log) [env:
+            LOGDNA_INCLUSION_RULES=]
+        --ip <ip>
+            The IP metadata to attach to lines forwarded from this agent [env: LOGDNA_IP=]
+        --journald-paths <journald-paths>...
+            List of paths (directories or files) of journald paths to monitor, for example: /var/log/journal or
+            /run/systemd/journal [env: LOGDNA_JOURNALD_PATHS=]
+    -k, --key <key>
+            The ingestion key associated with your LogDNA account [env: LOGDNA_INGESTION_KEY=]
+        --line-exclusion <line-exclusion>...
+            List of regex patterns to exclude log lines. When set, the Agent will NOT send log lines that match any of
+            these patterns [env: LOGDNA_LINE_EXCLUSION_REGEX=]
+        --line-inclusion <line-inclusion>...
+            List of regex patterns to include log lines. When set, the Agent will ONLY send log lines that match any of
```

Comment:
It's a tiny nit, but best to say "...the Agent will send ONLY log lines that match..." (It's typical to place the word "only" to be after the verb.)